platform/core/src/classes/metadata/StudyMetadata.js

```diff
@@ -818,6 +827,11 @@ const makeDisplaySet = (series, instances) => {
     imageSet.missingFrames = isReconstructable.missingFrames;
```

Comment:
isReconstructable feels like a boolean but its an object with multiple properties. Maybe we should rename it to something more declarative?

Reply:
you are right, new variable name:  displayReconstructableInfo